backend/tracim_backend/app_models/validator.py

```diff
@@ -1,11 +1,93 @@
+import re
+import typing
+
+from marshmallow import ValidationError
+from marshmallow.validate import Length
+from marshmallow.validate import Regexp
 from marshmallow.validate import OneOf
-from tracim_backend.app_models.contents import content_type_list
+from marshmallow.validate import Range
+from marshmallow.validate import Validator
 
+from tracim_backend.app_models.contents import GlobalStatus
+from tracim_backend.app_models.contents import content_status_list
+from tracim_backend.app_models.contents import content_type_list
 # TODO - G.M - 2018-08-08 - [GlobalVar] Refactor Global var
 # of tracim_backend, Be careful all_content_types_validator is a global_var !
+from tracim_backend.exceptions import TracimValidationFailed
+from tracim_backend.models.auth import Profile
+from tracim_backend.models.auth import User
+from tracim_backend.models.data import ActionDescription
+from tracim_backend.models.data import UserRoleInWorkspace
 
-all_content_types_validator = OneOf(choices=[])
+class TracimValidator(object):
+
+    def __init__(self):
+        self.validators = {}
+        self.values = {}
+
+    def add_validator(self, field:str, value:str, validator:typing.Callable) -> bool:  # nopep8
+        if not value:
+            return False
+
+        self.validators[field] = validator
+        self.values[field] = value
+        return True
 
+    def validate_all(self):
```

Comment:
Missing type and doctrsing. Additional question: Do we really need return a True when it is ok ? I don't think so.

Reply:
That's just more convenient for testing.